### ISSUE TITLE
add support for server.address()

### ISSUE THREAD

**Issue by srghma** [purescript-node/purescript-node-http#28]
there is already support for this method in https://github.com/purescript-node/purescript-node-net/blob/1fd91c872ecff202ae86b848f2c8d054f7a751f3/src/Node/Net/Server.purs#L72-L88

string means ICP or pipe of socket

http.server inherits from net.server

https://nodejs.org/api/http.html#http_class_http_server

the question is can it return `String` if server is created using purescript-node-http?

i.e. should it be 

```
address :: Server -> Effect (Maybe (Either Address String))
```

or 

```
address :: Server -> Effect (Maybe Address)
```

**Comment by srghma** [comment]
oh, yes, it can, we support sockets 

https://github.com/purescript-node/purescript-node-http/blob/b18d054de436ad589d41acee0aca7f3bd6762aab/src/Node/HTTP.purs#L68

**Comment by JordanMartinez** [comment]
Fixed by #49.
